packages/Menu/README.md

```diff
@@ -0,0 +1,54 @@
+# @MESG-COMPONENTS/MENU
+
+## Install
+
+`npm install @mesg-components/menu` or `yarn add @mesg-components/menu`
+
+## Properties
+
+- **banner**: `String`, **Required**, -> _banner to display on the footer_
+- **items**: `Array of object`, **Optional**, -> _items to display on the footer_
```

Comment:
could you describe what kind of objects are needed here

Reply:
done.